sdk/eventhub/azure-eventhubs/azure/eventhub/_common.py

```diff
@@ -378,28 +321,27 @@ def try_add(self, event_data):
         :raise: :class:`ValueError`, when exceeding the size limit.
         """
         if event_data is None:
-            log.warning("event_data is None when calling EventDataBatch.try_add. Ignored")
+            _LOGGER.warning("event_data is None when calling EventDataBatch.try_add. Ignored")
```

Comment:
Why are we silently failing on `try_add`ing a `None` eventdata? I would expect this to raise?

Reply:
Remove the type checking